[bug] Compilation fails on Tauri v2.6.1 when `common-controls-v6` feature is disabled

### Describe the bug

Compilation fails on tauri v2.6.1 when common-controls-v6 feature is disabled. Successful after re-enabling

### Reproduction

```shell
npm create tauri-app@latest
npm install

cd src-tauri
cargo add tauri --no-default-features -F wry,compression,devtools

cd ..
npm run tauri build
```

### Expected behavior

Build success

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.26100 x86_64 (X64)
    ✔ WebView2: 137.0.3296.93
    ✔ MSVC: Visual Studio 生成工具 2022
    ✔ rustc: 1.88.0 (6b00bc388 2025-06-23)
    ✔ cargo: 1.88.0 (873a06493 2025-05-10)
    ✔ rustup: 1.28.2 (e4f3ad6f8 2025-04-28)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 22.16.0
    - pnpm: 10.12.3
    - npm: 10.9.2

[-] Packages
    - tauri 🦀: 2.6.1
    - tauri-build 🦀: 2.3.0
    - wry 🦀: 0.52.1
    - tao 🦀: 0.34.0
    - @tauri-apps/api : 2.6.0
    - @tauri-apps/cli : 2.6.1

[-] Plugins
    - tauri-plugin-opener 🦀: 2.4.0
    - @tauri-apps/plugin-opener : 2.4.0

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - bundler: Vite
```

### Stack trace

```text
error[E0271]: type mismatch resolving `<HSTRING as TypeKind>::TypeKind == CopyType`
    --> D:\Development\Rust\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\tauri-runtime-wry-2.7.0\src\dialog\windows.rs:36:9
     |
34   |       MessageBoxW(
     |       ----------- required by a bound introduced by this call
35   |         None,
36   |         err,
     |         ^^^ expected `CopyType`, found `CloneType`
     |
     = note: required for `HSTRING` to implement `Param<windows_strings::pcwstr::PCWSTR, CopyType>`
note: required by a bound in `WindowsAndMessaging::MessageBoxW`
    --> D:\Development\Rust\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\windows-0.61.3\src\Windows\Win32\UI\WindowsAndMessaging\mod.rs:1571:9
     |
1569 | pub unsafe fn MessageBoxW<P1, P2>(hwnd: Option<super::super::Foundation::HWND>, lptext: P1, lpcaption: P2, ut...
     |               ----------- required by a bound in this function
1570 | where
1571 |     P1: windows_core::Param<windows_core::PCWSTR>,
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `MessageBoxW`

error[E0277]: the trait bound `HSTRING: Param<windows_strings::pcwstr::PCWSTR, CopyType>` is not satisfied
    --> D:\Development\Rust\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\tauri-runtime-wry-2.7.0\src\dialog\windows.rs:36:9
     |
34   |       MessageBoxW(
     |       ----------- required by a bound introduced by this call
35   |         None,
36   |         err,
     |         ^^^ the trait `Param<windows_strings::pcwstr::PCWSTR, CopyType>` is not implemented for `HSTRING`
     |
     = note: required for `HSTRING` to implement `Param<windows_strings::pcwstr::PCWSTR, CopyType>`
note: required by a bound in `WindowsAndMessaging::MessageBoxW`
    --> D:\Development\Rust\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\windows-0.61.3\src\Windows\Win32\UI\WindowsAndMessaging\mod.rs:1571:9
     |
1569 | pub unsafe fn MessageBoxW<P1, P2>(hwnd: Option<super::super::Foundation::HWND>, lptext: P1, lpcaption: P2, ut...
     |               ----------- required by a bound in this function
1570 | where
1571 |     P1: windows_core::Param<windows_core::PCWSTR>,
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `MessageBoxW`
help: consider borrowing here
     |
36   |         &err,
     |         +

Some errors have detailed explanations: E0271, E0277.
For more information about an error, try `rustc --explain E0271`.
error: could not compile `tauri-runtime-wry` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
failed to build app: failed to build app
```

### Additional context

_No response_

--- a/.changes/compile-without-common-controls.md
+++ b/.changes/compile-without-common-controls.md
@@ -1,0 +1,5 @@
+---
+tauri-runtime-wry: 'patch:bug'
+---
+
+Fixed a compile error when using tauri without the `common-controls-v6` feature enabled.

--- a/crates/tauri-runtime-wry/src/dialog/windows.rs
+++ b/crates/tauri-runtime-wry/src/dialog/windows.rs
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
-use windows::core::{w, HSTRING, PCWSTR};
+use windows::core::{w, HSTRING};
 
 enum Level {
   Error,
@@ -33,7 +33,7 @@ fn dialog_inner(err: &str, level: Level) {
     unsafe {
       MessageBoxW(
         None,
-        err,
+        &err,
         title,
         match level {
           Level::Warning => MB_ICONWARNING,

--- a/crates/tauri-runtime-wry/src/dialog/windows.rs
+++ b/crates/tauri-runtime-wry/src/dialog/windows.rs
@@ -46,7 +46,7 @@ fn dialog_inner(err: &str, level: Level) {
 
   #[cfg(feature = "common-controls-v6")]
   {
-    use windows::core::HRESULT;
+    use windows::core::{HRESULT, PCWSTR};
     use windows::Win32::Foundation::*;
     use windows::Win32::UI::Controls::*;
     use windows::Win32::UI::Shell::*;

